README.md

```diff
@@ -56,186 +50,416 @@ npm install @dojo/has
 npm install @dojo/shim
 npm install @dojo/core
 npm install @dojo/i18n
-npm install maquette
 ```
 
 You can also use the [dojo cli](https://github.com/dojo/cli) to create a complete Dojo 2 skeleton application.
 
 ## Features
 
-Constructing your own widgets (Custom widgets) is simple and straightforward.
-The smallest `@dojo/widget-core` example looks like this:
+### Basic Widgets
+
+Dojo 2 applications use a concept called Virtual DOM (vdom) to represent what should be shown on the screen. These vdom nodes are plain javascript objects that are not expensive to create, unlike browser DOM elements. Dojo 2 uses these vdom elements to synchronize and update the browser DOM so that application shows the expected view.
```

Comment:
Virtual DOM (vdom) should eventually link to https://github.com/dojo/dojo.io/blob/master/site/source/tutorials/working_with_virtual_dom.md when it is added to the dojo.io website